recipes/varlink/meta.yaml

```diff
@@ -0,0 +1,41 @@
+{% set name = "varlink" %}
+{% set version = "30.1.2" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "a4d8f28c60e1392dbfb61acf2dc7eb7d1c49ce9bf7f20615d9ada2a175e08acc"
+
+build:
+  number: 0
+  script: "{{ PYTHON }} -m pip install . --no-deps --ignore-installed -vv "
+
+requirements:
+  host:
+    - future  # [py27]
+    - pip
+    - python
+  run:
+    - future  # [py27]
+    - python
+
+test:
+  imports:
+    - varlink
+
+about:
+  home: "https://github.com/varlink/python"
+  license: "Apache Software"
+  license_family: "APACHE"
+  license_file: "LICENSE"
+  summary: "Varlink"
+  doc_url: "https://varlink.github.io/python/"
+  dev_url: "https://github.com/varlink/python/"
+
+extra:
+  recipe-maintainers:
+    - h-vetinari
+    - 4383
```

Comment:
@h-vetinari, I think conda-smithy doesn't like an integer here. Can you open an issue in https://github.com/conda-forge/conda-smithy ?

Reply:
Btw, try quoting here. `- "4383"`